### ISSUE TITLE
Material design

### ISSUE THREAD

**Issue by moshpirit** [jdeslip/arxiv-mobile#13]
Could you please include this style into the app? 
Thank you so much!! 😊

**Comment by jdeslip** [comment]
I agree! The UI needs a makeover. But, I personally don't have a lot of time these days. I'll see if I can scrounge up some time in the not to distant future, but pull requests are welcome.

Sorry for delay! I think when I imported the project from Google Code, my account was set to ignore this repo. So, I missed your message.

**Comment by jdeslip** [comment]
I've started a material design branch, which also includes a whole bunch of general code modernization for recent versions of Android. I expect to have something ready by years end with a beta likely available in October/November time frame.
